cg/meta/upload/fohm/fohm.py

```diff
@@ -227,16 +228,22 @@ def send_mail_reports(self) -> None:
             )
             if self._dry_run:
                 return
-            send_mail(
-                EmailInfo(
-                    receiver_email=self.config.fohm.email_recipient,
-                    sender_email=self.config.fohm.email_sender,
-                    smtp_server=self.config.fohm.email_host,
-                    subject=file.name,
-                    message=" ",
-                    file=file,
+
+            try:
+                send_mail(
+                    EmailInfo(
+                        receiver_email=self.config.fohm.email_recipient,
+                        sender_email=self.config.fohm.email_sender,
+                        smtp_server=self.config.fohm.email_host,
+                        subject=file.name,
+                        message=" ",
+                        file=file,
+                    )
                 )
-            )
+                file.unlink()  # Delete report
```

Comment:
```suggestion
                file.unlink()
```
Only inline comment when describing something very complex and hard to understand. Otherwise it just clutters.